common/src/main/proto/TSOProto.proto

```diff
@@ -16,51 +16,51 @@
 // limitations under the License.
 //
 
+syntax = "proto3";
```

Comment:
any reason for building with proto3? hbase still uses proto2 even though builds with proto 4.8.2?